src/textField/index.tsx

```diff
@@ -32,14 +32,16 @@ export interface CommonFormFields {
   ) => void
 }
 
+type errorField = string | JSX.Element | boolean
+
 export interface TextFieldProps extends CommonFormFields {
   isTextArea?: boolean
   defaultValue?: string
   labelledBy?: string
   onChange?: (obj: OnChangeParameters) => void
   onClear?: () => void
   className?: Classcat.Class
-  error?: string | JSX.Element
+  error?: errorField
```

Comment:
Shouldn't the boolean behavior be explained in the changelog and visible in the story?

Reply:
Boolean removed, I guess this is a legacy from a previous implementation of the Textfield... Weird, good catch!